.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -22,9 +22,23 @@ body:
     id: repro
     attributes:
       label: How can we reproduce it (as minimally and precisely as possible)?
+      description: |
+        Please list steps someone can follow to trigger the issue.
+        
+        For example:
+        1. Run `helm install mychart ./path-to-chart -f values.yaml`
```

Comment:
Could we add `--debug` by default ? I think we often require more details logs ?